modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js

```diff
@@ -4090,9 +4090,32 @@ AUI.add(
 				},
 
 				moveField(parentField, oldIndex, newIndex) {
+					var instance = this;
+
 					var fields = parentField.get('fields');
 
 					fields.splice(newIndex, 0, fields.splice(oldIndex, 1)[0]);
+
+					var fieldInstance = fields[newIndex];
+
+					var field = fieldInstance.getFieldDefinition();
+
+					if (field) {
+						var type = field.type;
+
+						if (type === 'ddm-text-html') {
+							var editor = fieldInstance.getEditor();
+
+							var usingAlloyEditor =
+								editor.getNativeEditor()._editor &&
+								editor.getNativeEditor()._editor.window.$
+									.AlloyEditor;
+
+							if (!usingAlloyEditor) {
```

Comment:
Hi @rolandpakai,

I believe I understood your logic here, if the HTML field isn't using AlloyEditor, then it'll be using CKEditor for sure, am I right? If so, it's important to say that other editors can be used besides AlloyEditor and CKEditor, please check the `portal.properties` [here](https://github.com/liferay/liferay-portal/blob/75ef5c682a28289697ec013e4580dd2a1a017808/portal-impl/src/portal.properties#L5558-L5576). In light of this, wouldn't be better create a logic which recreates the editor only when it's CKEditor?

Reply:
Hi @natocesarrego 

thanks for reviewing. Unfortunately i was not able to find any attribute/property/object which is obviously define the used editor is the CKEditor.

The only thing i was able to found in our code is this, and i created my solution based on it:
https://github.com/liferay/liferay-portal/blob/master/modules/apps/adaptive-media/adaptive-media-image-js-web/src/main/resources/META-INF/resources/ckeditor/plugins/adaptivemedia/plugin.js#L165

If you can provide a better solution i appreciate your help on it.

Thanks, Roland

Reply:
I see your point, @rolandpakai. Let's do the following: I'll close this pull request and we reopen the PTR associated to this issue for we find out a solution to address only the CKEditor use case. I'm suggesting we reopen the PTR, because there is a chance of we need to talk to the Front-end SME and to make this conversation happen would be better use the official way, e.g., the PTR project.